src/main/java/io/quarkus/search/app/SearchService.java

```diff
@@ -5,26 +5,30 @@
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.inject.Inject;
 import jakarta.transaction.Transactional;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
 import jakarta.ws.rs.DefaultValue;
 import jakarta.ws.rs.GET;
 import jakarta.ws.rs.Path;
 import jakarta.ws.rs.Produces;
 import jakarta.ws.rs.core.MediaType;
 
-import io.quarkus.search.app.dto.GuideSearchHit;
-import io.quarkus.search.app.dto.SearchResult;
-import io.quarkus.search.app.entity.Guide;
-import io.quarkus.search.app.entity.Language;
-import io.quarkus.search.app.entity.VersionAndLanguageRoutingBinder;
+import org.eclipse.microprofile.openapi.annotations.Operation;
 
 import org.hibernate.Length;
 import org.hibernate.search.engine.search.common.BooleanOperator;
 import org.hibernate.search.engine.search.predicate.dsl.SimpleQueryFlag;
 import org.hibernate.search.mapper.orm.session.SearchSession;
+import org.hibernate.validator.constraints.Range;
 
-import org.eclipse.microprofile.openapi.annotations.Operation;
 import org.jboss.resteasy.reactive.RestQuery;
 
+import io.quarkus.search.app.dto.GuideSearchHit;
+import io.quarkus.search.app.dto.SearchResult;
+import io.quarkus.search.app.entity.Guide;
+import io.quarkus.search.app.entity.Language;
+import io.quarkus.search.app.entity.VersionAndLanguageRoutingBinder;
```

Comment:
this seems to be not in the order defined in the pom... not that I mind 😄 but I suppose we are only having a sort/format part but not the check part in the build so we may miss it if something is not formatted ?

Reply:
... Maybe? Feel free to have a look :D